home/templates/home/home_page.html

```diff
@@ -32,9 +32,9 @@ <h2>{{ block.value.title }}</h2>
     {{page.search_header|richtext}}
     <form action="{% url 'results' %}" method="get">
         {{page.course_field_label}} <br>
-        <input id="courseName" type="text" name="courseQuery" {% if course_query %} value="{{ course_query }}"{% endif %} placeholder="Course Name" required ><br>
+        <input id="courseName" type="text" name="courseQuery" {% if course_query %} value="{{ course_query }}"{% endif %} placeholder="Course Name" required><br>
         {{page.institution_field_label}} <br>
-        <input id="instituionName" type="text" name="institutionQuery" {% if institution_query %} value="{{ institution_query }}"{% endif %} placeholder="Institution"><br>
+        <input id="instituionName" type="text" name="institutionQuery" {% if institution_query %} value="{{ institution_query }}"{% endif %} placeholder="Institution" required><br>
```

Comment:
So only one of the search terms should be required.

Reply:
Yes, at least one is required so both are required until one of the fields receive an input, then the required attribute for the field that has not received an input is remove. The logic for this is in main.js.